site/app/controllers/course/CourseMaterialsController.php

```diff
@@ -695,9 +695,14 @@ public function ajaxUploadCourseMaterialsFiles(): JsonResponse {
                 }
             }
 
-            $max_size = Utils::returnBytes(ini_get('upload_max_filesize'));
+            // Retrieve the max size allowed from the course config
+            // In CourseMaterialsController.php
```

Comment:
Instead of reading this file directly everytime, the system config has a model: https://github.com/Submitty/Submitty/blob/main/site/app/models/Config.php

You should add an entry here for the course materials upload file limit. You can also perform the checking if the key is not present then default to 100 mb.

That way you can do something like 
```
$core->getConfig()->getCourseMaterialsMaxFileUploadSize();
```